src/control/cmd/dmg/pool.go

```diff
@@ -83,6 +84,19 @@ func (d *PoolDestroyCmd) Execute(args []string) error {
 	return poolDestroy(d.log, d.conns, d.Uuid, d.Force)
 }
 
+// PoolGetACLCmd represents the command to fetch an Access Control List of a
+// DAOS pool.
+type PoolGetACLCmd struct {
+	logCmd
+	connectedCmd
+	UUID string `short:"p" long:"pool" required:"1" description:"UUID of DAOS pool"`
```

Comment:
I didn't provide a short alias for uuid on purpose to provide consistency with "daos" tool, after discussing with Ken. I've been asked to focus on syntax consistency.

Reply:
Good catch -- I didn't notice this.